### PR TITLE
Feat/xd 108

### PR DESCRIPTION
**Pull request by HaruspexSan**
added the headline in the header - needs some doing:
- add all tilte and subtitle data in each shell.route file
- add that the back button disappears on the homepage
- remove previous title and subtle

### CODE REVIEW

**Comment by HaruspexSan** [comment]
I added the missing titles, removed the previous titles and make the back button disappear on the home page

File: apps/frontend/src/app/components/header/header.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<ix-application class="h-screen">
+<ix-application class="h-screen z-0">
```
Comment (Sungonogi) [inline apps/frontend/src/app/components/header/header.component.html]: is that necesssary?

File: apps/frontend/src/app/components/header/header.component.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export interface IBreadcrumbData {
 @Component({
 	selector: 'app-header',
 	standalone: true,
-	imports: [ CommonModule, IxModule, RouterLink, RouterOutlet ],
+	imports: [CommonModule, IxModule, RouterLink, RouterOutlet],
```
Comment (Sungonogi) [inline apps/frontend/src/app/components/header/header.component.ts]: run
```
nx-run many -t lint
```
to fix the formatting

File: libs/cases/frontend/shell/src/lib/shell.routes.ts [hunk 3]
```diff
@@ -47,6 +51,8 @@ export const CASES_SHELL_ROUTES: Route[] = [
 						label: 'Details',
 						url: 'cases/:handle',
 					},
+					title: 'See the details of case',
```
Comment (Sungonogi) [inline libs/cases/frontend/shell/src/lib/shell.routes.ts]: Before these changes the title was
"Detail of Case"

**Review by Sungonogi — CHANGES_REQUESTED** [review]
Only found small things since we already worked on it yesterday

**Review by Sungonogi — APPROVED** [review]

**Pull request merged**